rs/ethereum/cketh/minter/src/eth_rpc_client/mod.rs

```diff
@@ -54,21 +52,23 @@ impl EthRpcClient {
     }
 
     pub fn from_state(state: &State) -> Self {
+        use evm_rpc_client::RpcServices as EvmRpcServices;
+
         let mut client = Self::new(state.ethereum_network());
         if let Some(evm_rpc_id) = state.evm_rpc_id {
             const MIN_ATTACHED_CYCLES: u128 = 500_000_000_000;
 
             let providers = match client.chain {
-                EthereumNetwork::Mainnet => EthereumProvider::evm_rpc_node_providers(),
-                EthereumNetwork::Sepolia => SepoliaProvider::evm_rpc_node_providers(),
+                EthereumNetwork::Mainnet => EvmRpcServices::EthMainnet(None),
+                EthereumNetwork::Sepolia => EvmRpcServices::EthSepolia(None),
             };
             let min_threshold = match client.chain {
                 EthereumNetwork::Mainnet => 3_u8,
                 EthereumNetwork::Sepolia => 2_u8,
             };
             let threshold_strategy = EvmRpcConfig {
                 response_consensus: Some(ConsensusStrategy::Threshold {
-                    total: None,
```

Comment:
Do you think it would make sense to introduce a constant instead of using `4` in the source code? It may then be a bit clearer what the number means (and also slightly easier to update).

Reply:
Good idea, that's also allow us to have test ensuring that this number is the same as the number of providers used by the minter directly (useful in case we need to revert to not using the EVM RPC canister, in which case we should have similar trust assumptions)